2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Configure stats pid to 19688
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Loading settings from C:\Users\loreb\.config\wandb\settings
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Loading settings from c:\Users\loreb\Documents\IntellijProjects\PycharmProjects\DLA\Lab1\wandb\settings
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_init.py:_log_setup():527] Logging user logs to c:\Users\loreb\Documents\IntellijProjects\PycharmProjects\DLA\Lab1\wandb\run-20240429_194334-vhzr7b7u\logs\debug.log
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_init.py:_log_setup():528] Logging internal logs to c:\Users\loreb\Documents\IntellijProjects\PycharmProjects\DLA\Lab1\wandb\run-20240429_194334-vhzr7b7u\logs\debug-internal.log
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_init.py:_jupyter_setup():473] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000017F5A6C1F90>
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_init.py:init():567] calling init triggers
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {'architecture': 'ResNet18', 'dataset': 'CIFAR10', 'epochs': 20, 'learning_rate': 0.001, 'batch_size': 128, 'device': device(type='cpu'), 'optimizer': 'Adam'}
2024-04-29 19:43:34,390 INFO    MainThread:19688 [wandb_init.py:init():592] re-initializing run, found existing run on stack: zsbaqnrn
2024-04-29 19:43:34,398 INFO    MainThread:19688 [wandb_run.py:_finish():2052] finishing run lorebaia/DLA Assigment 1/zsbaqnrn
2024-04-29 19:43:34,398 INFO    MainThread:19688 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-29 19:43:34,398 INFO    MainThread:19688 [wandb_run.py:_restore():2275] restore
2024-04-29 19:43:34,398 INFO    MainThread:19688 [wandb_run.py:_restore():2281] restore done
2024-04-29 19:43:40,231 INFO    MainThread:19688 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-29 19:43:40,231 INFO    MainThread:19688 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-29 19:43:40,238 INFO    MainThread:19688 [wandb_run.py:_footer_sync_info():3879] logging synced files
2024-04-29 19:43:40,318 INFO    MainThread:19688 [wandb_init.py:init():617] starting backend
2024-04-29 19:43:40,318 INFO    MainThread:19688 [wandb_init.py:init():621] setting up manager
2024-04-29 19:43:40,330 INFO    MainThread:19688 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-29 19:43:40,334 INFO    MainThread:19688 [wandb_init.py:init():629] backend started and connected
2024-04-29 19:43:40,369 INFO    MainThread:19688 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-29 19:43:40,369 INFO    MainThread:19688 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-29 19:43:40,369 INFO    MainThread:19688 [wandb_init.py:init():721] updated telemetry
2024-04-29 19:43:40,464 INFO    MainThread:19688 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-04-29 19:43:42,230 INFO    MainThread:19688 [wandb_run.py:_on_init():2344] communicating current version
2024-04-29 19:43:42,952 INFO    MainThread:19688 [wandb_run.py:_on_init():2353] got version response upgrade_message: "wandb version 0.16.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-29 19:43:42,952 INFO    MainThread:19688 [wandb_init.py:init():805] starting run threads in backend
2024-04-29 19:43:50,052 INFO    MainThread:19688 [wandb_run.py:_console_start():2323] atexit reg
2024-04-29 19:43:50,052 INFO    MainThread:19688 [wandb_run.py:_redirect():2178] redirect: wrap_raw
2024-04-29 19:43:50,052 INFO    MainThread:19688 [wandb_run.py:_redirect():2243] Wrapping output streams.
2024-04-29 19:43:50,052 INFO    MainThread:19688 [wandb_run.py:_redirect():2268] Redirects installed.
2024-04-29 19:43:50,052 INFO    MainThread:19688 [wandb_init.py:init():848] run started, returning control to user process
2024-04-29 19:44:01,772 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:44:01,772 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:57:47,942 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:57:47,958 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:57:47,958 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:57:54,808 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:57:54,818 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:57:54,818 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:57:59,381 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:57:59,389 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:57:59,389 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:58:54,111 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:58:54,119 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:58:54,119 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:59:07,594 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:59:07,604 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:59:07,604 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:59:11,569 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:59:11,569 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:59:11,569 INFO    MainThread:19688 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-29 19:59:15,928 INFO    MainThread:19688 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-29 19:59:16,061 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-29 19:59:16,066 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Configure stats pid to 19688
2024-04-29 19:59:16,066 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Loading settings from C:\Users\loreb\.config\wandb\settings
2024-04-29 19:59:16,066 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Loading settings from c:\Users\loreb\Documents\IntellijProjects\PycharmProjects\DLA\Lab1\wandb\settings
2024-04-29 19:59:16,066 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-29 19:59:16,066 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_init.py:_log_setup():527] Logging user logs to c:\Users\loreb\Documents\IntellijProjects\PycharmProjects\DLA\Lab1\wandb\run-20240429_195916-pnkajlro\logs\debug.log
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_init.py:_log_setup():528] Logging internal logs to c:\Users\loreb\Documents\IntellijProjects\PycharmProjects\DLA\Lab1\wandb\run-20240429_195916-pnkajlro\logs\debug-internal.log
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_init.py:init():567] calling init triggers
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {'architecture': 'ResNet18', 'dataset': 'CIFAR10', 'epochs': 20, 'learning_rate': 0.001, 'batch_size': 128, 'device': device(type='cpu'), 'optimizer': 'Adam'}
2024-04-29 19:59:16,074 INFO    MainThread:19688 [wandb_init.py:init():592] re-initializing run, found existing run on stack: vhzr7b7u
2024-04-29 19:59:16,077 INFO    MainThread:19688 [wandb_run.py:_finish():2052] finishing run lorebaia/DLA Assigment 1/vhzr7b7u
2024-04-29 19:59:16,077 INFO    MainThread:19688 [jupyter.py:save_history():445] not saving jupyter history
2024-04-29 19:59:16,077 INFO    MainThread:19688 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-29 19:59:16,077 INFO    MainThread:19688 [wandb_init.py:_jupyter_teardown():455] cleaning up jupyter logic
2024-04-29 19:59:16,077 INFO    MainThread:19688 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-29 19:59:16,077 INFO    MainThread:19688 [wandb_run.py:_restore():2275] restore
2024-04-29 19:59:16,077 INFO    MainThread:19688 [wandb_run.py:_restore():2281] restore done
2024-04-29 19:59:22,416 INFO    MainThread:19688 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-29 19:59:22,416 INFO    MainThread:19688 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-29 19:59:22,428 INFO    MainThread:19688 [wandb_run.py:_footer_sync_info():3879] logging synced files
